components/Frame/Header.js

```diff
@@ -366,31 +368,41 @@ class Header extends Component {
               height={this.state.mobile ? HEADER_HEIGHT_MOBILE : HEADER_HEIGHT}
             />
           )}
-          <Popover expanded={!!expand} inNativeApp={inNativeApp} me={me} isMobile={mobile}>
-            <NavPopover
-              me={me}
-              url={url}
-              inNativeApp={inNativeApp}
-              closeHandler={this.close}
-            />
-          </Popover>
         </div>
-        {!inNativeApp && me && opaque && (
-          <NavBar
+        {isMember && opaque && (
+          <Fragment>
+            <hr
+              {...styles.stickyWithFallback}
+              {...styles.hr}
+              {...styles.hrThin} />
+            <NavBar fixed={withoutSticky} url={url} />
+          </Fragment>
+        )}
+        <hr
+          {...styles[isMember ? 'sticky' : 'stickyWithFallback']}
+          {...((isMember && withoutSticky && styles.hrFixedAfterNavBar) || undefined)}
+          {...styles.hr}
+          {...styles[formatColor ? 'hrThick' : 'hrThin']}
+          style={formatColor ? {
+            color: formatColor,
+            backgroundColor: formatColor
+          } : undefined} />
+        <Popover expanded={!!expand} inNativeApp={inNativeApp}>
+          <NavPopover
+            me={me}
             url={url}
-            onNavBarChange={onNavBarChange}
-            mobile={mobile}
-            formatColor={formatColor}
-            sticky={navbarSticky}
-            initial={navbarInitial || expand}
+            inNativeApp={inNativeApp}
+            closeHandler={this.close}
           />
-        )}
+        </Popover>
         <LoadingBar />
-        {!!cover && !inline && <div {...styles.cover}>{cover}</div>}
-
-      </div>
+        {!!cover && <div {...styles.cover}>{cover}</div>}
```

Comment:
On pages with a cover (e.g. /jobs) the thin HR is visible when the header is hidden:
<img width="1299" alt="screen shot 2018-08-23 at 11 23 11" src="https://user-images.githubusercontent.com/23520051/44517546-4e2c5b80-a6c8-11e8-9368-cb348132429f.png">

Reply:
✅ fixed